CONTRIBUTING.md

```diff
@@ -419,19 +419,45 @@ The Chapter client uses the React framework [Next.js](https://nextjs.org/) with
 After you have added new feature, to make sure it stays working, we recommend using [Cypress](https://www.cypress.io/). It will automatically test different scenarios starting in the client side of the application, communicating with the server, and warning you if something unexpected happens.
 </details>
 
-## Where to Find the Code
+## Where to Find the Code to Change
 
+### Database schema
```

Comment:
Nit pick. Suggest to capitalize "Schema" here for consistency with other H3 headings.